plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1631,6 +1631,12 @@ void producer_plugin_impl::plugin_startup() {
       }
 
       if (_ro_thread_pool_size > 0) {
+#if defined(EOSIO_EOS_VM_RUNTIME_ENABLED) || defined(EOSIO_EOS_VM_JIT_RUNTIME_ENABLED)
+         // inform controller's wasm_alloc_pool of number of updated total threads
+         // (number of main plus read only threads)
+         chain.set_wasm_alloc_pool_num_threads(1 + _ro_thread_pool_size);
```

Comment:
It is a no-op to call this, seems like it would be clearer to always call it.

Reply:
Looks like you missed this. Not a big deal.

Reply:
It was removed by https://github.com/AntelopeIO/spring/pull/1346/commits/267018ee044dbe52953c000fdad89723ecb567f5 (at the bottom of that commit)

Reply:
I meant that I think it would be cleaner to call even if `_ro_thread_pool_size == 0`

Reply:
Got it. Will do it in https://github.com/AntelopeIO/spring/pull/1378. Thanks.